plugins/discourse-ai/lib/completions/endpoints/open_ai.rb

```diff
@@ -14,7 +14,7 @@ def normalize_model_params(model_params)
           # max_tokens is deprecated however we still need to support it
           # on older OpenAI models and older Azure models, so we will only normalize
           # if our model name starts with o (to denote all the reasoning models)
-          if llm_model.name.starts_with?("o")
+          if llm_model.name.starts_with?("o") | llm_model.name.starts_with?("gpt-5")
```

Comment:
Then that magic const can be used here as well

Reply:
Cool yeah will follow up next week